pkg/virt-controller/watch/vm/vm_test.go

```diff
@@ -6259,6 +6259,59 @@ var _ = Describe("VirtualMachine", func() {
 				})
 			})
 
+			Context("Tolerations", func() {
```

Comment:
We should have a test case for:
1. add tolerations to none
2. add tolerations to some
3. remove some from some
4. remove all from some